MainModule/Client/UI/Aero.rbxmx

```diff
@@ -299,7 +297,7 @@ return function(data, env)
 	frame:TweenSize(UDim2.new(1, -5, 0, 60),'Out','Quad',0.2)
 	frame:TweenPosition(UDim2.new(0,0,1,-75),'Out','Linear',0.2)
 
-	task.spawn(function()
+	spawn(function()
```

Comment:
im fairly certain that spawn is not overwritten with task.spawn inside the theme env